static/js/initCardMapView.js

```diff
@@ -121,4 +53,94 @@ function clearMarkers() {
         markers[i].setMap(null);
     }
     markers = [];
-}
\ No newline at end of file
+}
+
+
+if (navigator.geolocation) {
```

Comment:
is this part of production code? if not, let's remove

Reply:
What's mean?